spec/std/html_spec.cr

```diff
@@ -66,5 +66,16 @@ describe "HTML" do
       str = HTML.unescape("limit &#1114111;")
       str.should eq("limit 􏿿")
     end
+
+    it "unescapes &NotSquareSuperset;" do
+      str = HTML.unescape(" &NotSquareSuperset; ")
+
+      str.should eq(" ⊐̸ ")
+    end
+
+    it "unescapes &ampd" do
```

Comment:
typo: `&ampd` -> `&amp`

Reply:
I don't think it's a typo. At least not in that sense. It was copied from my comment.
If at all, it should be changed to `unescapes &amphello`.